Src/AutoFixtureUnitTest/Kernel/FiniteSequenceRequestTest.cs

```diff
@@ -19,20 +19,19 @@ public void InitializeWithNullRequestAndValidNumberThrows()
             // Teardown
         }
 
-        [Theory]
-        [InlineData(0)]
-        [InlineData(-1)]

```

Comment:
I agree that it would be good with more than one test value, because it would protect against a Devil's Advocate implementation that specifically checks for `-1` only.